### DISCUSSION TITLE
Docker & MacOS

### DISCUSSION THREAD

**Discussion by arieunier** [allo-/virtual_webcam_background#67] [Q&A]
Hello there, 
i'd be keen to test this on a mac using docker. Before spending time on it, do you if this has already been done in the past by some of the users ? And if so, with which results ?

**Comment by allo-** [comment]
You will need a video looback driver. When there is one on MacOS it might work. 

When you get this to work on MacOS: https://atinkerholic.wordpress.com/2018/10/10/how-to-use-a-virtual-webcam-with-static-image-or-video-ffmpeg-v4l-utils-and-v4l2loopback/

It should be possible to get the program to work.

**Comment by kirk86** [comment]
Hey @arieunier,

> Hello there, I'd be keen to test this on a mac using docker. Before spending time on it, do you if this has already been done in the past by some of the users ? And if so, with which results ?

if you're still interested this does work on MacOS, for a detailed step by step guide have a look [here](https://kirk86.github.io/2022/02/virtual-webcam-background/) on how to run it on OSX.
